rust/candid/tests/assets/ok/class.d.ts

```diff
@@ -1,7 +1,6 @@
 import type { Principal } from '@dfinity/agent';
-import type BigNumber from 'bignumber.js';
 export type List = [] | [
-  [BigNumber, List]
+  [BigInt, List]
```

Comment:
In spots like this, the type would be `bigint`, which means we expect an instance of a BigInt

```suggestion
  [bigint, List]
```